bookings/components/booking-filters.yaml

```diff
@@ -0,0 +1,464 @@
+    format: date
+  example: '2019-03-01'
+  description: >-
+    Filters on bookings that have a follow up email scheduled to be sent on or before 
+    the given date.
+has_preorders:
+  name: has_preorders
+  in: query
+  required: false
+  schema:
+    type: boolean
+  description: >-
+    Filters on bookings that have preorders.
+labels:
+  name: labels
+  in: query
+  required: false
+  explode: false
+  schema:
+    type: array
+    items:
+      type: string
+  example:
+    - Need to call
+  description: >-
+    Filters on bookings that have the given labels. If a comma-separated list is 
+    passed, bookings which have any of the given labels will be returned.
+last_updated:
+  name: last_updated
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-01-01'
+  description: >-
+    Filters on bookings which were last updated on the given date.
+last_updated_from:
+  name: last_updated_from
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-01-01'
+  description: >-
+    Filters on bookings which were last updated on or after the given date.
+last_updated_to:
+  name: last_updated_to
+  in: query
+  required: false
+  schema:
+    type: string
+    format: date
+  example: '2019-01-03'
+  description: >-
+    Filters on bookings which were last updated on or before the given date.
+notifications:
+  name: notifications
+  in: query
+  required: false
+  explode: false
+  schema:
+    type: array
+    items:
+      type: string
+  example:
+    - new_messages
+  description: >-
+    Filters on bookings which have the given notification. If a comma-separated list is 
+    passed, bookings which have all of the given notifications will be returned.
+num_people:
+  name: num_people
+  in: query
+  required: false
+  schema:
+    type: string
+  description: >-
+    Filters on bookings which have the given number of guests. You can optionally pass through 
+    a range, for example `6 TO 10` would filter on bookings containing between 6 and 10 guests.
+  example: 10
+offer:
+  name: offer
+  in: query
+  required: false
+  explode: false
+  schema:
+    type: array
+    items:
+      type: string
+  description: >-
+    Filters on bookings which have the given offer. This filter accepts a single offer ID, or multiple 
+    offer IDs separated by a comma to filter on bookings containing any of the given offers. To filter 
+    on bookings with *no* offer, use `none`.
+partner_source:
+  name: partner_source
+  in: query
+  required: false
+  schema:
+    type: string
+  example: 'christmas_page'
+  description: >-
+    Filters on bookings which have the given `partner_source`.
+pending_deposits:
+  name: pending_deposits
+  in: query
+  required: false
+  explode: false
+  schema:
+    type: array
+    items:
+      type: string
+  description: >-
+    Filters on bookings which have a pending deposit of the given type. If a comma-separated 
```

Comment:
Do we need to describe the possible types?

Reply:
Done - have I missed any?

* request_auth
* manual_auth
* request_payment
* manual_payment
* customer_preorder

Reply:
There's also a `paid` type for deposits that were added as paid outside of Collins.